ui/app/pages/send/send-content/add-recipient/ens-input.component.js

```diff
@@ -182,7 +182,7 @@ export default class EnsInput extends Component {
   renderSelected () {
     const { t } = this.context
     const { className, selectedAddress, selectedName, addressBook } = this.props
-    const contact = addressBook.filter(item => item.address === selectedAddress)[0] || {}
+    const contact = addressBook.filter(item => item.address.toLowerCase() === selectedAddress.toLowerCase())[0] || {}
```

Comment:
Would it be possible to normalize these addresses somewhere instead of converting them to lower-case for comparisons? This was done in a few other places too. It seems like a code smell :thinking: .

Reply:
Addressed in https://github.com/MetaMask/metamask-extension/pull/6978/commits/a267a797f76c9be792dcc58f2e5bbd027d091b0a